### PR TITLE
cp-304: Card view with long titles. Fix

### PR DESCRIPTION
**Pull request by OlesiaTr**

### CODE REVIEW

File: frontend/src/libs/components/card/styles.module.scss [hunk 1]
```diff
@@ -72,6 +71,12 @@
   word-break: break-word;
 }
 
+@media (width >= 900px) {
```
Comment (liza-veis) [inline frontend/src/libs/components/card/styles.module.scss]: we use < sign everywhere, so here we should use it too

**Review by liza-veis — CHANGES_REQUESTED** [review]

File: frontend/src/libs/components/card/styles.module.scss (hunk 1 above)
Comment (OlesiaTr) [reply frontend/src/libs/components/card/styles.module.scss]: Would it be ok if I put 899 then? Because without = it will be broken on 900px

Comment (liza-veis) [reply frontend/src/libs/components/card/styles.module.scss]: you can use =, but we need to invert the sign <= will be ok

Comment (OlesiaTr) [reply frontend/src/libs/components/card/styles.module.scss]: Ohhh, but can I use strict measures then?

Comment (OlesiaTr) [reply frontend/src/libs/components/card/styles.module.scss]: I can write it like this

`@media (900px <= width) `

And it's working, but it feels so wrong😖. And I used to write it more like this `@media (min-width: 900px)`

Comment (liza-veis) [reply frontend/src/libs/components/card/styles.module.scss]: I also like this option `@media (min-width: 900px)`, but we use this `@media (900px <= width)` in other places in the app

Comment (OlesiaTr) [reply frontend/src/libs/components/card/styles.module.scss]: Ye, I was asking cause didn't see in the development this way of writing it yet. Ok, I changed it!

**Review by OlenaYatseniuk — DISMISSED** [review]

**Review by liza-veis — APPROVED** [review]

**Review by v0ldemar01 — DISMISSED** [review]

File: frontend/src/libs/components/card/styles.module.scss [hunk 2]
```diff
@@ -73,6 +72,12 @@
   word-break: break-word;
 }
 
+@media (900px <= width) {
```
Comment (liza-veis) [inline frontend/src/libs/components/card/styles.module.scss]: we need to join it with a query below for < 900px

**Review by liza-veis — CHANGES_REQUESTED** [review]

**Comment by OlesiaTr** [comment]
With changes looks like this

https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/104642736/c9f99e45-6617-4c57-8c11-fae3641be9a6

**Review by dmytro-kucherenko — APPROVED** [review]

**Review by OlenaYatseniuk — APPROVED** [review]

**Review by v0ldemar01 — APPROVED** [review]

**Review by fshabanov — APPROVED** [review]

**Review by liza-veis — APPROVED** [review]

**Pull request merged**